chroot-bin/croutonclip

```diff
@@ -42,9 +42,9 @@ copyclip() {
             # Check if display is still running
             if rundisplay "$current" xdpyinfo >/dev/null 2>&1; then
                 echo -n 'R'
-                rundisplay "$current" xclip -o -sel clip
+                rundisplay "$current" xsel -ob
             else
-                echo -n "EUnable to open display '$current'."
+                echo -n "Unable to open display '$current'."
```

Comment:
Actually, this wasn't a typo.  Error messages sent to the extension are prefixed with E